variables.tf

```diff
@@ -247,9 +247,9 @@ variable "saml_role_assertion" {
 ################################################################################
 
 variable "associate_license" {
-  description = "Determines whether a license will be associated with the workspace"
+  description = "Determines whether a license will be associated with the workspace. Use false if you do not have an active enterprise license in the AWS Marketplace."
   type        = bool
-  default     = true
```

Comment:
we should maintain the current default otherwise this is considered a breaking change

Reply:
Understood